qa/TL1_git_submodule/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+test_body() {
+    export TMP_PATH="$(mktemp -d)"
+    cd ..
+    export DALI_REPO_DIR=$(pwd)
+    pushd $TMP_PATH
+    git init
+    git submodule add $DALI_REPO_DIR
```

Comment:
Correction. I think you need to update our test infra to copy `.git` folder into the test docker image.

Reply:
why? We are creating a git repo on the fly here. No need to copy anything

Reply:
Because you are adding the DALI repo as a submodule. And we need to make DALI code a repo. In our CI we just copy the source code to our test env, bit not the .git directory.